docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -442,6 +442,27 @@ include::{generated-dir}/config/quarkus-vertx-http-config-group-access-log-confi
 Use `quarkus.http.access-log.exclude-pattern=/some/path/.*` to exclude all entries concerning the path `/some/path/...` (_including subsequent paths_) from the log.
 ====
 
+== Arbitrary customizations
+
+Quarkus allows users to arbitrarily customize the options of HTTP servers started by Quarkus via the use of `io.quarkus.vertx.http.HttpServerOptionsCustomizer`.
+For example, if the HTTP port needs to be set programmatically, then the following code could be used:
```

Comment:
Can be everything from https://vertx.io/docs/apidocs/io/vertx/core/http/HttpServerOptions.html configured or there are some limitations that Quarkus imposes?

Reply:
What kind of limitations are you thinking of? 

There isn't anything I am personally aware of

Reply:
That's pretty much implied, no?

Reply:
No limitations, but also... no safety belt...